src/main/webapp/app/exercises/modeling/participate/modeling-submission.service.ts

```diff
@@ -67,11 +68,13 @@ export class ModelingSubmissionService {
      * @param {number} exerciseId - Id of the exercise
      * @param {boolean?} lock - True if assessment is locked
      */
-    getModelingSubmissionForExerciseWithoutAssessment(exerciseId: number, lock?: boolean): Observable<ModelingSubmission> {
-        let url = `api/exercises/${exerciseId}/modeling-submission-without-assessment`;
+    getModelingSubmissionForExerciseForCorrectionRoundWithoutAssessment(exerciseId: number, lock?: boolean, correctionRound?: number): Observable<ModelingSubmission> {
+        correctionRound = correctionRound ? correctionRound : 0;
+        let url = `api/exercises/${exerciseId}/round/${correctionRound}/modeling-submission-without-assessment`;
         if (lock) {
             url += '?lock=true';
         }
+        console.log('url', url);
```

Comment:
Do we want to have a log here or was it for debugging purposes?

Reply:
It was for debugging